src/components/FeaturedProducts/index.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Card from "../common/Card";
+import StyledFlex from "../common/StyledFlex";
+import ImageContainer from "../common/Image";
+
+const FeaturedProductsGrid = ({featuredProducts, isLoading}) => {
```

Comment:
It's not a good practice for code change when the tests are created.

Reply:
Sure, just did it , because I hadn't done your suggestions before because of the hurry of the deadline.